### DISCUSSION TITLE
[Question][Hw1] module function 'int_of_bool' of 'Value' module not being declared on its signature

### DISCUSSION THREAD

**Discussion by bscho99** [prosyslab-classroom/cs424-program-reasoning#271] [Q&A]
Name: Cho Beomsuk

Hello, this might be unimportant but I discovered while solving HW1 that the 'int_of_bool' function defined on 'Value' module is absent on its signature.
I wonder if it is a minor mistake made during source code write-up, so it is up to student to add that in the signature,
or if it is intended and thus not allowed to add it.

**Comment by goodtaeeun** [comment]
Hi, we do not require you to add the signature of `int_of_bool`. Feel free to either ignore it or add the signature yourself.
